nucypher/utilities/prometheus/metrics.py

```diff
@@ -156,7 +156,12 @@ def create_metrics_collectors(ursula: "lawful.Ursula") -> List[MetricsCollector]
     collectors: List[MetricsCollector] = [UrsulaInfoMetricsCollector(ursula=ursula)]
 
     # Blockchain prometheus
-    collectors.append(BlockchainMetricsCollector(eth_endpoint=ursula.eth_endpoint))
+    collectors.append(
+        BlockchainMetricsCollector(
+            root_net_endpoint=ursula.eth_endpoint,
```

Comment:
`root` and `child` seem like the wrong titles - we know the node is going to have them set correctly otherwise it won't be participating in rituals.

From a monitoring standpoint, it's going to be more useful to know __which__ chains i'm connected to (if we add extra chains conditions, all nodes will need to have a new rpc_